src/modules/add-and-remove.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/prefer-default-export */
+import { Task } from './to-to';
+import { input } from './html-element';
+// js for add and delete functionalities
+
+class functionalty {
+  static addNewTask() {
+    // js to create a new task
+    // js to setup local storage
+    if (localStorage.getItem('myTasks') === null) {
+      localStorage.setItem('myTasks', '[]');
+    }
+
+    const newTaskDescription = input.value;
+    const checked = false;
+    const taskslist = JSON.parse(localStorage.getItem('myTasks'));
+    const newTaskId = taskslist.length;
+
+    const task = new Task(newTaskDescription, checked, newTaskId);
+
+    // getting old data and merging with new data
```

Comment:
- [ ] Please, consider that I could not find the Editing functionality, so you have to implement it also, as it is required in this milestone 👍

Reply:
ok